lmnet/lmnet/datasets/bdd100k.py

```diff
@@ -28,6 +27,25 @@ class BDD100K(ObjectDetectionBase):
     available_subsets = ["train", "validation"]
     extend_dir = "bdd100k"
 
+    @property
+    def label_colors(self):
+        bike = [119, 11, 32]
+        bus = [0, 60, 100]
+        car = [0, 0, 142]
+        motor = [0, 0, 230]
+        person = [220, 20, 60]
+        rider = [255, 0, 0]
+        traffic_light = [250, 170, 30]
+        traffic_sign = [220, 220, 0]
+        train = [0, 80, 100]
+        truck = [0, 0, 70]
+
+        return np.array([
+            bike, bus, car, motor,
+            person, rider, traffic_light,
+            traffic_sign, train, truck
+        ])
+
     @classmethod
     @functools.lru_cache(maxsize=None)
     def count_max_boxes(cls, base_path=None):
```

Comment:
Now, this function is not necessary.

Reply:
The function `count_max_boxes` is there for conventional purposes. As you dig deeper sometimes, this function is called from the base class e.g. in `pascalvoc_2007` dataset. I think we should keep this to ensure operation of the BDD100K dataset class throughout the project. If you go to other object detection dataset all of them have this function.

Reply:
I think this is not only a problem of BDD100K.  Just leave it as you say.